src/screens/DistrictMatch/fetchDistrict.js

```diff
@@ -19,9 +19,10 @@ export default async function(){
       }else{
         const officeName = resultJson.offices[0].name.split(' ');
         const district = officeName[officeName.length-1];
-        this.setState({district, view:'DISTRICT_FOUND'}, () => {
-          this.setState({isLoading: false});
-        });
+        /**
+         * Do something with the district here then navigate to main App...
```

Comment:
Yes please save the district in redux and Firebase!!

Reply:
I would prefer to merge this and have that work exist on a separate branch.